plugins/ern_v0.14.0+/react-native-fast-image_v5.3.0+/config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "com.android.support:support-v4:27.1.1",
```

Comment:
we will have to discuss with the team to agree on semantics to default to app wide `support-v4` version

Reply:
That's ok, done through https://github.com/electrode-io/electrode-native/pull/1116 
Thanks to the dedupe feature introduce in mentioned PR, only one version of support-v4 will be injected, and the version will be the one configured through `supportLibraryVersion` in Electrode Native configuration (or default one for specific electrode native version if not set). 
That being said, we should inject by default `com.android.support:support-v4` in the Container as it is done for `support-v7`. Will open a PR for that, so that it's not needed for plugins to add explicit dependency on `support-v4`/`support-v7` in plugin config.